docs/source/README.rst

```diff
@@ -1,27 +1,50 @@
 VOLTTRON
 ========
 
-VOLTTRON will solve your problem of where to start with documentation,
-by providing a basic explanation of how to do it easily.
+VOLTTRON(TM) is an open source platform for distributed sensing and control. The platform provides services
+for collecting and storing data from buildings and devices and provides an environment for developing applications
+which interact with that data.
 
-Look how easy it is to use:
-
-    import project
-    # Get your stuff done
-    project.do_stuff()
 
 Features
 --------
 
-- Be awesome
-- Make things faster
+- Driver framework for collecting data from and sending control actions to buildings and devices
+- Historian framework for storing data
+- Agent lifecycle managment
 
 Installation
 ------------
 
 Install VOLTTRON by running:
 
-    install project
+    git clone https://github.com/VOLTTRON/volttron
+    cd volttron
+    python bootstrap.py
+
+This will build the platform and create a virtual Python environment. Activate this and then start the platform with:
+
+    . env/bin/activate
+    volttron -vv -l volttron.log&
+
+This enters the virtual Python environment and then starts the platform in debug (vv) mode with a log file named
+volttron.log.
+
+Next, start an example listener to see it publish and subscribe to the message bus:
+
+    scripts/core/make-listener
+
+This script handles several different commands for installing and starting an agent after removing an old copy. This
```

Comment:
I don't think this rendering is what you had in mind.
![index-error-rendering](https://cloud.githubusercontent.com/assets/3979063/19505632/4028977c-9579-11e6-8d01-1db32327b8e2.png)